frontend/src/sass/_themes.scss

```diff
@@ -0,0 +1,296 @@
+  unselected-chip:          $dark-bg-lighter-20,
+  disabled-list-option:     $dark-bg-lighter-10,
+);
+
+// Compute font config
+// @include mat.core($fontConfig); // had to change this to have no argument due to errors
+@include mat.core();
+
+// Theme Config
+body {
+  --primary-color: #544ac3;
+  --primary-lighter-color: #ccc9ed;
+  --primary-darker-color: #3a32ae;
+  --text-primary-color: #{$light-primary-text};
+  --text-primary-lighter-color: #{$dark-primary-text};
+  --text-primary-darker-color: #{$light-primary-text};
+}   
+$mat-primary: (
+  main: #544ac3,
+  lighter: #ccc9ed,
+  darker: #3a32ae,
+  200: #544ac3, // For slide toggle,
+  contrast : (
+    main: $light-primary-text,
+    lighter: $dark-primary-text,
+    darker: $light-primary-text,
+  )
+);
+$theme-primary: mat.define-palette($mat-primary, main, lighter, darker);
+
+// body {
+//   --secondary-color: #607d8b;
+//   --secondary-lighter-color: #cfd8dc;
+//   --secondary-darker-color: #3a4b53;
+//   --text-secondary-color: #{$light-primary-text};
+//   --text-secondary-lighter-color: #{$dark-primary-text};
+//   --text-secondary-darker-color: #{$light-primary-text};
+// }   
+// $mat-secondary: (
+//   main: #607d8b,
+//   lighter: #cfd8dc,
+//   darker: #3a4b53,
+//   200: #b0bec5, // For slide toggle,
+//   contrast : (
+//     main: $light-primary-text,
+//     lighter: $dark-primary-text,
+//     darker: $light-primary-text,
+//   )
+// );
+// $theme-secondary: mat.define-palette($mat-secondary, main, lighter, darker);
+
+body {
+  --accent-color: #00ff75;
+  --accent-lighter-color: #b3ffd6;
+  --accent-darker-color: #00ff58;
+  --text-accent-color: #{$dark-primary-text};
+  --text-accent-lighter-color: #{$dark-primary-text};
+  --text-accent-darker-color: #{$dark-primary-text};
+}   
+$mat-accent: (
+  main: #00ff75,
+  lighter: #b3ffd6,
+  darker: #00ff58,
+  200: #00ff75, // For slide toggle,
+  contrast : (
+    main: $dark-primary-text,
+    lighter: $dark-primary-text,
+    darker: $dark-primary-text,
+  )
+);
+$theme-accent: mat.define-palette($mat-accent, main, lighter, darker);
+
+body {
+  --warn-color: #f15946;
+  --warn-lighter-color: #fbcdc8;
+  --warn-darker-color: #eb3e2f;
+  --text-warn-color: #{$dark-primary-text};
+  --text-warn-lighter-color: #{$dark-primary-text};
+  --text-warn-darker-color: #{$light-primary-text};
+}   
+$mat-warn: (
+  main: #f15946,
+  lighter: #fbcdc8,
+  darker: #eb3e2f,
+  200: #f15946, // For slide toggle,
+  contrast : (
+    main: $dark-primary-text,
+    lighter: $dark-primary-text,
+    darker: $light-primary-text,
+  )
+);
+$theme-warn: mat.define-palette($mat-warn, main, lighter, darker);
+
+$httat-theme: (
+  color: (
+    primary: $theme-primary,
+    accent: $theme-accent,
+    warn: $theme-warn,
+    is-dark: false,
+    foreground: $mat-light-theme-foreground,
+    background: $mat-light-theme-background,
+  ),
+  typeography: $fontConfig,
+  density: 0,
+);
+
+// $theme: (
```

Comment:
These can be removed, rather than commented out.

Reply:
I removed the commented out the unneeded commented-out code.